tensorboard/webapp/widgets/histogram/histogram_component.ng.html

```diff
@@ -77,11 +70,17 @@
       </g>
     </g>
 
-    <g #histograms class="histograms">
+    <g
+      #histograms
+      [class.histograms]="true"
+      [class.linked-time-enabled]="linkedTime"
```

Comment:
is this class name used?

Reply:
It is unused in test or CSS but I thought it would be useful to have this class in general. One use case would be to use it in webdriver test.